### ISSUE TITLE
Add graphql_term_object_mutation_update_additional_data hook

### ISSUE THREAD

**Issue by tomgreeEn** [wp-graphql/wp-graphql#1429]
We currently have the ability to run an action after additional post data has been updated using 'graphql_post_object_mutation_update_additional_data'

However, an equivalent hook is not available to run an action after _term_ data has been updated - this should be added for consistency and completeness.

**Comment by renatonascalves** [comment]
Actually, you can use this: https://github.com/wp-graphql/wp-graphql/blob/df4cbef058874514285a12393cd9c0469649680a/src/Mutation/TermObjectUpdate.php#L155

**Comment by renatonascalves** [comment]
I agree that how it works currently is not consistent but this hook can be "hooked" on term update.

**Comment by justlevine** [comment]
@renatonascalves @tomgreeEn , I'm combing through old tickets for easy candidates. 

From my understanding, the only tangible differences between `graphql_{type}_mutation_update_additional_data` and `graphql_{update|insert}_term` actions is that the former:
- the mutation `$input` is available to the action, and
- `..._update_additional_data` runs on both create and update mutations, while for terms you'd need to hook the function to _both_ `graphql_update_term` and `graphql_insert_term` to get the same results.

If so, it should be easy enough add a `TermObjectMutation::update_additional_term_object_data()` function that does nothing inherently but call a comparable filter.

@jasonbahl with your approval, I'd be happy to submit a PR.

**Comment by renatonascalves** [comment]
I personally don't mind it. But if we **really** want to make things consistent, I'd rather we confirm how other mutations are doing this so that we don't introduce another edge case here and instead update this everywhere if/when needed.

> consistency and completeness.

Not even WordPress is this consistent and complete. So if we decide to keep it as is since you can use that other hook, I'm also okay with it, but I agree it might be weird or odd for newcomers. :)

**Comment by justlevine** [comment]
bad stalebot
